src/gradata/detection/__init__.py

```
@@ -0,0 +1,24 @@
+)
+
+__all__ = [
+    "classify_mode",
+    "MODE_CATEGORY_MAP",
+    "is_addition",
+    "classify_addition",
+    "AdditionTracker",
+    "detect_conflict",
+    "extract_diff_tokens",
+    "ConflictTracker",
+]
```

Ruff (0.15.9)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)